kalliope/brain.yml

```diff
@@ -14,3 +14,15 @@
       - say:
           message:
             - "Hello sir"
+
+  - name: "default-synapse"
+    signals:
+      - order: "default-synapse-order"
+    neurons:
+      - say:
+          message:
+            - "Je n'ai pas compris vôtre ordre"
+            - "Je ne connais pas cet ordre"
```

Comment:
As the whole current settings is configured to use FR, it's consistent :).
Do I switch everything to English?